Java/Java_evenodd_example.java

```diff
@@ -0,0 +1,10 @@
+import java.util.Scanner;
+//This code tells whether a given number is even or odd
+public class Java_evenodd_example {
+    public static void main(String[] args) {
+        Scanner sc = new Scanner(System.in);
+        System.out.println("Enter a number: ");
+        int num = sc.nextInt();
+        System.out.println(num % 2 == 0 ? "The number is even" : "The number is odd");
+    }

```

Comment:
Always close the Scanner object by .close() method before ending the program, in long run it would be beneficial to avoid memory leakage.

Reply:
Oh ok! Really didn't see that. Thanks for pointing that out.